Makefile

```diff
@@ -1,83 +1,38 @@
 .DEFAULT_GOAL := help
-SHELL := /bin/bash
+NPM           ?= npm
+SHELL         := /bin/bash
 
-DOCKER_BUILDER_IMAGE_NAME = govuk/notify-nodejs-client-runner
-
-BUILD_TAG ?= notifications-nodejs-client-manual
-
-DOCKER_CONTAINER_PREFIX = ${USER}-${BUILD_TAG}
-
-.PHONY: help
 help:
 	@cat $(MAKEFILE_LIST) | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: dependencies
-dependencies:  ## Install build dependencies
-	npm install
-
-.PHONY: build
-build: dependencies ## Build project
-
-.PHONY: test
-test: ## Run tests
-	npm test
+ci:
+	$(NPM) ci --production
 
-.PHONY: integration-test
-integration-test: ## Run integration tests
-	npm test --integration
+ci-dev:
+	$(NPM) ci
 
-.PHONY: generate-env-file
-generate-env-file: ## Generate the environment file for running the tests inside a Docker container
-	script/generate_docker_env.sh
+install:
+	$(NPM) install --production
 
-.PHONY: prepare-docker-runner-image
-prepare-docker-runner-image: ## Prepare the Docker builder image
-	make -C docker build
+install-dev:
+	$(NPM) install
 
-.PHONY: build-with-docker
-build-with-docker: prepare-docker-runner-image ## Build inside a Docker container
-	docker run -i --rm \
-		--name "${DOCKER_CONTAINER_PREFIX}-build" \
-		-v "`pwd`:/var/project" \
-		-e http_proxy="${HTTP_PROXY}" \
-		-e HTTP_PROXY="${HTTP_PROXY}" \
-		-e https_proxy="${HTTPS_PROXY}" \
-		-e HTTPS_PROXY="${HTTPS_PROXY}" \
-		-e NO_PROXY="${NO_PROXY}" \
-		${DOCKER_BUILDER_IMAGE_NAME} \
-		make build
+build:
+	$(NPM) run build ## Build project
```

Comment:
Can just do `npm build` if you want

Reply:
Seems to return a warning and doesn't execute: `npm WARN build `npm build` called with no arguments. Did you mean to `npm run-script build`?`
[See this from SO](https://stackoverflow.com/questions/29939697/npm-build-doesnt-run-the-script-named-build-in-package-json)